src/main/resources/info/trainee.txt

```diff
@@ -0,0 +1,5 @@
+Mariam Katamashvili 2004-01-01 Tbilisi 1
+Mariam Katamashvili 2005-01-01 Kutaisi 2
+John Doe 2000-01-01 Batumi 3
+John Doe 2001-01-01 Poti 4
+Mariam Katamashvili 2005-01-01 Kutaisi 5
```

Comment:
Would be more close to the reality with import of CSV. Or Excel format but it requires more processing.